src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,34 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { userId, title, body, user, comments } = post;
```

Comment:
I would move every key to the new line for better readability 
```suggestion
  const { 
    userId, 
    title, 
    body, 
    user, 
    comments 
  } = post;
```